audio_to_spectrogram/scripts/spectrum_plot.py

```diff
@@ -35,10 +57,16 @@ def _cb(self, msg):
         self.ax.set_xlim((self.freq.min(), self.freq.max()))
         self.ax.set_ylim((0.0, 20))
         self.ax.legend(loc='upper right')
+        print(self.pub_img.get_num_connections())
```

Comment:
You may forget to remove this line?

If you need this line, `rospy.loginfo` or `rospy.logdebug` seems better.

Reply:
Thanks. I forgot to remove the debug print.
Fixed.